### PR TITLE
[RpmVPackages] Add 'chrony' package verification

### PR DESCRIPTION
**Pull request by psachin**
Signed-off-by: Sachin Patil <psachin@redhat.com>

### CODE REVIEW

**Comment by psachin** [comment]
@xiangce Can you review this please. The tests are not needed as such but I still included them to make sure everything works fine.

File: insights/parsers/rpm_v_packages.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """
-RpmVPackages - command ``/bin/rpm -V coreutils procps procps-ng shadow-utils passwd sudo``
-==========================================================================================
+RpmVPackages - command ``/bin/rpm -V coreutils procps procps-ng shadow-utils passwd sudo chrony``
```
Comment (xiangce) [inline insights/parsers/rpm_v_packages.py:2]: How about just change it as 

```
RpmVPackages - command ``/bin/rpm -V <package names>``
```

So, we do not need to update it whenever adding new packages.

File: insights/specs/insights_archive.py [hunk 2]
```diff
@@ -243,7 +243,7 @@ class InsightsArchiveSpecs(Specs):
     root_crontab = simple_file("insights_commands/crontab_-l_-u_root")
     rndc_status = simple_file("insights_commands/rndc_status")
     route = simple_file("insights_commands/route_-n")
-    rpm_V_packages = simple_file("insights_commands/rpm_-V_coreutils_procps_procps-ng_shadow-utils_passwd_sudo")
+    rpm_V_packages = simple_file("insights_commands/rpm_-V_coreutils_procps_procps-ng_shadow-utils_passwd_sudo_chrony")
```
[the remark this answers is not in the document]
Comment (psachin) [reply insights/specs/insights_archive.py:246]: I agree. Very good catch. :100:

File: insights/parsers/rpm_v_packages.py (hunk 1 above)
Comment (psachin) [reply insights/parsers/rpm_v_packages.py]: @xiangce Sure. My concern was that the user will have no idea which packages are verified. He will always have to refer the specs. To void this, I have include the list in the docstring.

Comment (psachin) [reply insights/parsers/rpm_v_packages.py]: Done.

**Comment by psachin** [comment]
> @psachin - This change relates to a SPEC change to the uploader.json, an INSPEC card is needed. Please create an INSPEC Jira card and modify the uploader.json as well.

https://github.com/RedHatInsights/insights-core-assets/pull/406/

**Pull request merged**